internal/v1/handler.go

```diff
@@ -1215,3 +1215,69 @@ func closeBody(body io.Closer) {
 		logrus.Errorf("closing response body failed: %v", err)
 	}
 }
+
+func (h *Handlers) GetOscapProfiles(ctx echo.Context, distribution Distributions) error {
+	switch distribution {
+	case Rhel8:
+		fallthrough
+	case Rhel84:
+		fallthrough
+	case Rhel85:
+		fallthrough
+	case Rhel86:
+		fallthrough
+	case Rhel87:
+		fallthrough
+	case Rhel88:
```

Comment:
So technically, I think anything past  RHEL 8.6 and RHEL 9.1 can take on OpenSCAP customizations.

Reply:
Do you think it's worth changing the switch case for something else ?

Reply:
Since the deployment has the most recent version of composer, wouldn't rhel84/5 with openscap work as well?

Reply:
Yeah it will work too

Reply:
All done :)